werkit/aws_lambda/test_parallel.py

```diff
@@ -0,0 +1,107 @@
+import pytest
+import pdb
+import asyncio
+import json
+
+import werkit.aws_lambda.parallel
+from botocore.exceptions import ClientError
+
+from functools import partial
+
+import io
+
+from werkit.aws_lambda.test_worker.service import handler as worker_handler
+
+from werkit.aws_lambda.test_util import (
+    inputs,
+    extra_args,
+    lambda_worker_function_name,
+    parallel_map_on_lambda_timeout_failure_call_worker_service_mock,
+    setup_first_failure_mock_responses,
+    setup_success_mock_responses,
+    setup_mock_failure_response,
+    default_timeout,
+)
+
+from asynctest import Mock, patch
+
+
+@patch("aioboto3.client", new_callable=Mock)
+def test_call_worker_service_success(mock_invoke):
+    # test that we expect to get a successful response out
+    # stubber = init()
```

Comment:
rm?

Reply:
Fixed in 5cbf2f38d0cebf4240abbc12753f29b27fc8e014